plutus-list/plutus-list.cabal

```diff
@@ -0,0 +1,72 @@
+cabal-version:      3.0
+name:               plutus-list
+version:            1.0
+extra-source-files: CHANGELOG.md
+
+common lang
+  default-language:   Haskell2010
+  default-extensions:
+    BangPatterns
+    BinaryLiterals
+    ConstraintKinds
+    DataKinds
+    DeriveFunctor
+    DeriveGeneric
+    DeriveTraversable
+    DerivingStrategies
+    DerivingVia
+    DuplicateRecordFields
+    EmptyCase
+    FlexibleContexts
+    FlexibleInstances
+    GADTs
+    GeneralizedNewtypeDeriving
+    HexFloatLiterals
+    ImportQualifiedPost
+    InstanceSigs
+    KindSignatures
+    LambdaCase
+    MultiParamTypeClasses
+    NumericUnderscores
+    OverloadedStrings
+    ScopedTypeVariables
+    StandaloneDeriving
+    TupleSections
+    TypeApplications
+    TypeOperators
+    TypeSynonymInstances
+    UndecidableInstances
+
+  build-depends:      base ^>=4.14
+  ghc-options:
+    -Wall -Wcompat -Wincomplete-uni-patterns -Wredundant-constraints
+    -Wmissing-export-lists -Werror -Wincomplete-record-updates
+    -Wmissing-deriving-strategies
+
+library
+  import:          lang
+  exposed-modules:
+    PlutusTx.List.Natural
+    PlutusTx.List.Ord
+
+  build-depends:
+    , playground-common
+    , plutus-numeric     ^>=2.3
+    , plutus-tx
+    , template-haskell   ^>=2.16.0.0
+
+  hs-source-dirs:  src
+
+test-suite plutus-list-test
```

Comment:
This should be named something slightly more meaningful, since we're likely to have multiple test suites. If this is a suite of property tests, `properties` is fine.